shared/actions/wallets.js

```diff
@@ -216,6 +217,16 @@ const requestPayment = (state: TypedState) =>
     })
   )
 
+const startPayment = () =>
+  RPCStellarTypes.localStartBuildPaymentLocalRpcPromise().then(bid =>
+    WalletsGen.createBuildingPaymentIDReceived({bid})
+  )
+
+const stopPayment = (state: TypedState, action: WalletsGen.StopPaymentPayload) =>
+  RPCStellarTypes.localStopBuildPaymentLocalRpcPromise({bid: state.wallets.building.bid}).then(_ =>
+    WalletsGen.createClearBuilding()
```

Comment:
Would it work without this `createClearBuilding`?

If `stopBuildPaymentLocal` was slow and a user was unreasonable quick to open a new form, seems like this clearBuilding action could come in asynchronously and break their new form. Maybe the stopPayment action should clear the store orthogonal to the stop RPC. In the reducer stopPayment could do the same thing as clearBuilding.

Reply:
@mlsteele makes sense, done